python/pyiceberg/io/pyarrow.py

```diff
@@ -120,6 +124,10 @@
 ONE_MEGABYTE = 1024 * 1024
 BUFFER_SIZE = "buffer-size"
 ICEBERG_SCHEMA = b"iceberg.schema"
+FIELD_ID = "id"
```

Comment:
This should be `field_id`, right?
```suggestion
FIELD_ID = "field_id"
```